### PR TITLE
Added Béchamel sauce

### PR DESCRIPTION
**Pull request by michaelpande**

### CODE REVIEW

**Comment by michaelpande** [comment]
Tested, tastes good :+1:

File: white_sauce.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# White sauce / Béchamel sauce
+#### Easy, relible and low stress way of making the original Béchamel sauce
```
Comment (olehermanse) [inline white_sauce.md]: Spelling: reliable

Comment (olehermanse) [inline white_sauce.md]: I don't like that you're using a header (`<h4>`) for this. Plain text, or an `*italic*` is more appropriate.

File: white_sauce.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Teaspoon
+- Measuring cup (250 ml or more)
+
+##### Time
+Ready in 10 minutes
+
+##### People
+1 person, multiply if more.
+
+##### Ingredients you should have available
```
Comment (olehermanse) [inline white_sauce.md]: Should be simplifed to just `Ingredients`

File: white_sauce.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+##### Good to know
+- The actual amounts of each ingredient you should actually add is listed below.
+- People have different tea-/tablespoons
+- It's better to use less heat and more time, the result is the same.
+
+
+
+## 1
+Melt 50g of butter in a pot / boiler. Make sure you don't burn it.
+Then move the pot from the heat source
```
Comment (olehermanse) [inline white_sauce.md]: Missing full stop (period).

File: white_sauce.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+## 3.
+Add 250ml of milk, and stir until perfectly blended. It will seem to be too liquid, but will solidify until perfect consistency with heat
+
+## 4
+- Move pot to heat source and stir on medium heat until you have your preferred consistency.
+- Move your pot from the heat source.
+
+## 5 - Be careful!
+Very carefully add more salt by tasting.
+This will make or break your sauce. If you start getting uncertain, stop adding more salt.
```
Comment (olehermanse) [inline white_sauce.md]: You have 2 sentences on one line, add a newline after the period.

File: white_sauce.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## 2
+- Add 4 flat tablespoons of flour and stir until completely blended.
+Now you carefully have to add enough flour spoon by spoon until it
+is just past the point of being liquid. This should not require much more than an additional 1-2 tablespoons
+- Add 1/4th teaspoon of salt. Regardless of salted or unsalted butter.
+- Add 1/4th teaspoon of pepper
+- Add 1/6th teaspoon of nutmeg
+
+## 3.
```
Comment (olehermanse) [inline white_sauce.md]: You don't have a dot in the other numbers, remove.

**Comment by michaelpande** [comment]
@olehermanse Updated PR

**Pull request merged**